packages/coordinator/src/services/validation/bundle-validator.ts

```diff
@@ -529,3 +539,18 @@ function checkPrimaryCarePrescriptionResources(
     return errors.missingRequiredField("organization.partOf")
   }
 }
+
+function validateTelecoms(bundle: fhir.Bundle, incorrectValueErrors: Array<fhir.OperationOutcomeIssue>) {
+  const telecoms = getTelecoms(bundle)
+
+  telecoms.forEach(telecom => validateTelecom(telecom, incorrectValueErrors))
+}
+
+function validateTelecom(telecom: fhir.ContactPoint, incorrectValueErrors: Array<fhir.OperationOutcomeIssue>) {
+  if(!telecom.value){
+    incorrectValueErrors.push(errors.missingRequiredField("telecom.value"))
+  }
+  if(!telecom.use){
+    incorrectValueErrors.push(errors.missingRequiredField("telecom.use"))
```

Comment:
These aren't hugely helpful as they don't say which telecom is wrong.

Reply:
85fece397ccebd4c7cc55d9e55edf10067ea6f0a
b5da8e4fe633d4f720216294a8f040adb4465b32